.github/workflows/test.yml

```
@@ -91,3 +91,29 @@ jobs:
       - name: Run browser tests
         working-directory: ./acvm_js
         run: yarn test:browser
+
+  test-acvm_js-rust:
+    name: Rust Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu
+            runner: ubuntu-latest
+            target: wasm32-unknown-unknown
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@1.67.0
```

```suggestion
        uses: dtolnay/rust-toolchain@1.66.0
```

We should test on our MSRV